client/my-sites/current-site/notice.jsx

```diff
@@ -303,6 +339,8 @@ export default connect(
 			isMigrationInProgress,
 			hasJITM: getTopJITM( state, messagePath ),
 			messagePath,
+			siteOptions: getSiteOptions( state, siteId ),
+			siteSlug,
```

Comment:
Just noting that it looks like the `site`, which has a `slug` property is already in the props and being used by [some other nudges](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/current-site/notice.jsx#L187).

Reply:
👍